proposals/nnnn-configurable-time-before-shutdown.md

```diff
@@ -0,0 +1,77 @@
+# Configurable time before shutdown
+
+* Proposal: [SDL-NNNN](nnnn-configurable-time-before-shutdown.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+To prevent missing logs after SDL shutdown, additional ini file options should be added : 
+ - Write all logs to file system before shutdown 
+ - Maximum time of SDL shutting down
+ 
+## Motivation
+
+SDL produces a tone of logs in the asynchronous mode in some use cases (like video streaming or big put file),
+SDL collects these logs in queue and write them to file system in a separate thread.
+Writing to the file system may require big amount of time.
+
+In current implementation after receiving IGNITION_OFF signal SDL drops all logs that was not written yet.
```

Comment:
"In the current implementation, after receiving IGNITION_OFF signal, SDL drops all logs that have not yet been written."